### PR TITLE
Band-aiding this heap of sadness that's facehuggers.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Applying some rapid tweaks and adjustments to reduce facehuggers inconsistency and RNG. I'd call this band-aid because I now believe they need a subsystem to handle this at a more appropriate tick rate than once every 2 seconds, but can't do right away.

## Why It's Good For The Game
Reducing facehuggers inconsistency and RNG a little. Super optimal outcome not granted.

## Changelog
:cl:
tweak: Tweaking facehuggers once more in preparation of a possibly more appropriate subsystem for them.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/facehuggers.dm [hunk 1]
```diff
@@ -234,27 +235,25 @@
 	update_icon()
 
 /obj/item/clothing/mask/facehugger/throw_impact(atom/hit_atom, speed)
-	if(stat == DEAD)
+	if(stat != CONSCIOUS)
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/facehuggers.dm]:
```suggestion
	if(stat)
```

Comment (DominikPanic) [reply code/modules/mob/living/carbon/xenomorph/facehuggers.dm]: While technically true it's MUCH less readable.

Comment (Ghommie) [reply code/modules/mob/living/carbon/xenomorph/facehuggers.dm]: ultimately, this is why the CONSCIOUS define even exists.

Comment (Rohesie) [reply code/modules/mob/living/carbon/xenomorph/facehuggers.dm]: I prefer `if(stat)` myself, but this is not something we should be enforcing. `if(stat != CONSCIOUS)` is fine.

**Pull request merged**